cli/tests/cmd.rs

```diff
@@ -661,7 +661,7 @@ contract BTest is DSTest {
 });
 
 // test that `forge build` does not print `(with warnings)` if there arent any
-forgetest!(can_compile_without_warnings, |prj: TestProject, mut cmd: TestCommand| {
+forgetest_ignore!(can_compile_without_warnings, |prj: TestProject, mut cmd: TestCommand| {
```

Comment:
Why is this ignored? :thinking:

Reply:
ignored the wrong test, smh, should have been the spells